lib/config/author_config.yml

```diff
@@ -0,0 +1,41 @@
+aji:
+  link: https://thoughtbot.com/blog/authors/aji-slater
+
+dimiter:
+  link: https://thoughtbot.com/blog/authors/dimiter-petrov
+
+elisa:
+  link: https://github.com/elisuh
+
+fer:
+  link: https://github.com/ferperales
+
+jeanine:
+  link: https://thoughtbot.com/blog/authors/jeanine-soterwood
+
+matheusrich:
+  link: https://thoughtbot.com/blog/authors/matheus-richard
+
+mburns:
+  link: https://thoughtbot.com/blog/authors/mike-burns
+
+neilson:
+  link: https://thoughtbot.com/blog/authors/neil-carvalho
+
+remy:
+  link: https://thoughtbot.com/blog/authors/remy-hannequin
+
+csara:
+  link: https://thoughtbot.com/blog/authors/sara-jackson
```

Comment:
nit: This seems to be the only one not alphabetically sorted (if the goal was to sort by slackname)